README.md

```diff
@@ -11,6 +11,9 @@ Feel free to contribute: [issues](https://github.com/dkhamsing/open-source-ios-a
 - Edhita: Text editor
   - https://github.com/tnantoka/edhita
   - https://itunes.apple.com/app/edhita-open-source-text-editor/id398896655
+- Jupp: App with share extension for ADN
+  - https://github.com/dasdom/Jupp
+  - https://itunes.apple.com/de/app/jupp-share-extension-for-app.net/id909926740
 - HN Reader: Hacker News Reader
   - https://github.com/Dimillian/SwiftHN
   - https://itunes.apple.com/app/hn-reader-hacker-news-reader/id919243741
```

Comment:
visitBCN - visit countries/detail 
https://github.com/maurovc/visitBCN
https://itunes.apple.com/us/app/visitbcn/id904676442?l=es&ls=1&mt=8
a menjar - mobile restaurant app
https://github.com/maurovc/aMenjar
https://itunes.apple.com/us/app/a-menjar!/id816473131?l=es&ls=1&mt=8
Color Blur -blur/edit your pic
https://github.com/maurovc/ColorBlur
https://itunes.apple.com/us/app/id928863510
Peggsite - collage your pics in a storyboard
https://github.com/jenduf/GenericSocialApp
https://itunes.apple.com/us/app/peggsite/id938445951?mt=8
iGrades - track your grades/ calculate
https://github.com/maurovc/iGrades
https://itunes.apple.com/us/app/id816987574